src/targets/gpu/jit/pointwise.cpp

```diff
@@ -167,8 +187,11 @@ struct pointwise_compiler : compiler<pointwise_compiler>
         auto name = g.create_function(
             g.generate_module(*pm).set_attributes({"__device__"}).set_generic_types(*pm));
         std::string lambda = "MIGRAPHX_LIFT(" + name + ")";
+        auto op_names      = get_op_names(*pm);
```

Comment:
Do the `join_strings` here and pass the kernel name in.

Reply:
fixed